src/capture.js

```diff
@@ -439,11 +439,16 @@ Capture.ios8AndGreaterScrollFix = function(doc, callback) {
     // Using `getElementsByTagName` here because grabbing head using
     // `document.head` will throw exceptions in some older browsers (iOS 4.3).
     var head = doc.getElementsByTagName('head');
-    // Be extra safe and guard against `head` not existing.
-    if (!head.length) {
+    var viewport = doc.querySelector('meta[name="viewport"]');
+
+    // Be extra safe and guard against `head` not existing. We also don't need
+    // to do anything if the original source didn't have a viewport metatag.
```

Comment:
Somewhat confused - we still need to add the viewport tag into the page if there isn't one present... That's what forces the page to attempt to scroll. I believe what we talked about earlier is we said "if the page doesn't have a viewport in the original page, our injection of a viewport tag will cause the page to be broken if we unmobify, and thus, we should inject another viewport tag into the page at 960px when unmobifying, because that is the default, and will get the website back into a normal state"